src/src/LeftMenu/LeftMenu.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import MenuListItem from './MenuListItem/MenuListItem'
+
+const LeftMenu = () => {
+    return (
+        <StyledLeftMenu>
+            <MenuListItem className={'menuItem'}>Bookshelf</MenuListItem>
```

Comment:
I think you can actually write `className="menuItem"` and it will be more readable.

Reply:
I remade this part of code, and I'm not using className for styled-components, but I corrected all strings in attributes.